Source/WebCore/accessibility/AriaAttributes.idl

```diff
@@ -33,27 +33,27 @@ interface mixin AriaAttributes {
     [Reflect=aria_colcount]         attribute DOMString? ariaColCount;
     [Reflect=aria_colindex]         attribute DOMString? ariaColIndex;
     [Reflect=aria_colspan]          attribute DOMString? ariaColSpan;
-    [Reflect=aria_controls, EnabledBySetting=AriaReflectionForElementReferencesEnabled] attribute FrozenArray<Element>? ariaControlsElements;
+    [CustomGetter, Reflect=aria_controls, EnabledBySetting=AriaReflectionForElementReferencesEnabled] attribute FrozenArray<Element>? ariaControlsElements;
```

Comment:
It's unfortunate that we have to write a custom getter for all these properties.

Reply:
I don't think that's necessary for now. If we start deploying this kind of code elsewhere, we can start thinking about it.